pkg/rule/rule.go

```diff
@@ -343,22 +344,22 @@ func executeCommandAction(
 			return err
 		}
 		// If env is not set get the values from stork deployment spec.
-		registry, registrySecret, err := k8sutils.GetImageRegistryFromDeployment(
+		registry, registrySecret, imageTag, err := k8sutils.GetImageInfoFromDeployment(
 			k8sutils.StorkDeploymentName,
 			storkPodNs,
 		)
 		if err != nil {
 			return err
 		}
 		if len(registry) != 0 {
-			cmdExecutorImage = registry + "/" + defaultCmdExecutorImage
+			cmdExecutorImage = registry + "/" + defaultCmdExecutorImage + ":" + imageTag
 		} else {
-			cmdExecutorImage = defaultCmdExecutorImage
+			cmdExecutorImage = defaultCmdExecutorImage + ":" + defaultCmdExecutorTag
 		}
 		cmdExecutorImageSecret = registrySecret
 	} else {
 		// if env is set get it from env variable.
-		cmdExecutorImage = os.Getenv(cmdExecutorImageRegistryEnvVar) + "/" + defaultCmdExecutorImage
+		cmdExecutorImage = os.Getenv(cmdExecutorImageRegistryEnvVar) + "/" + defaultCmdExecutorImage + ":" + defaultCmdExecutorTag
```

Comment:
Here 0.1 version of cmdexecutor is still being pulled.  I am assuming when cmdExecutorImageRegistryEnvVar is used then annotation cmdExecutorImageOverrideKey is used to override the image. If needed a new environment variable to override cmdexecutor image can be introduced but I am not sure how it will be set.

Reply:
I think we can stick with just the annotations for now. We should push for not using any of these env variables or annotations. If a registry is being used then the same registry will be used for both stork and cmdexecutor and we should just push for using the same tags.